static/js/script.js

```diff
@@ -0,0 +1,118 @@
+function ajaxPostCallRequest(url, data, callbackSuccess=null, callbackFailure=null){
+    $.ajax({
+        url: url,
+        method: "POST",
+        cache: false,
+        dataType: 'json',
+        headers: {
+            'Authorization': `Bearer ${window.localStorage.getItem('accessToken')}`
+        },
+        data: data,
+        success: function (data){
+            if (callbackSuccess){
+                callbackSuccess(data);
+            }
+        },
+        error: function (data){
+            if (callbackFailure){
+                callbackFailure(data);
+            }
+        }
+    })
+}
+// user registration
+$(document).ready(function (){
+    // id of the form for the submit
```

Comment:
multiple times $(document).ready(function (){ used in single file. move all DOM loaded content in one